### DISCUSSION TITLE
National Geographic HD East

### DISCUSSION THREAD

**Discussion by jphilippe2017** [iptv-org/community#744] [Q&A]
**Channel Name:** Nat Geo TV
**Channel ID:** NationalGeographicHDEast.us
**Country:** USA
**Language:** English
**Website:** https://www.natgeotv.com/hd
**Stream URL (optional):**
**Notes (optional):** HD please!

**Comment by freearhey** [comment]
Closed in favor of https://github.com/iptv-org/iptv/issues/19730
